CryptoPrices.xcodeproj/project.pbxproj

```diff
@@ -296,6 +297,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		1CF4DD4D2939B61B0070B0EE /* Run Swiftlint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Run Swiftlint";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif [ -z \"$CI\" ]; then\n  if which swiftlint > /dev/null; then\n    swiftlint\n  else\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\n  fi\nfi\n";
+		};
+/* End PBXShellScriptBuildPhase section */
+
```

Comment:
There're some notes about to integrate `swiftlint` into the project here 🙏 : 
- The `Run Swiftlint` build phase should be placed right after `CompileSources`, so it might help to validate our code a little faster while developing on our local machine.
  
  <img width="1128" alt="Screenshot 2022-12-02 at 13 53 13" src="https://user-images.githubusercontent.com/17972674/205235991-33c5136f-1491-4968-b6fd-3c4ec8d2c017.png">

- This setup assumes that the developer already installs `swiftlint` on his/her local machine (if not just a warning to install it). I think it might be better to have an instruction or quick setup from the beginning (when the developer first checkout source code) to do so. I suggest making a simple shell script like this-put in the `Makefile` (you don't want to add this script, adding the install instructions in the `README` is perfectly 👌 ):
  ```
  setup:
      brew install swiftlint
      bundle install --path vendor/bundle
  ```

  This script is meant to be run first after checking out the source code. With it in hand, you can replace all
  ```
  - name: Bundle install
    run: bundle install --path vendor/bundle
  ```
  in the workflows with:
  ```
  - name: Setup
    run: make setup
  ```
  (One of the other benefits is that it's much easier for non-developer like PM, Tester, ... just run only one `make setup` command than a bunch of others with long names and many parameters)
  You can apply this to other tasks too, e.g. `fastlane` tasks. Let's say you want to build and test the app on your machine, you have to open your terminal and then type quite a long command `bundle exec fastlane buildAndTest`. Instead you can add this to `Makefile` with a shorter name `buildAndTest`. Then on both the local machine and CI just `make buildAndTest`. But this is entirely optional, we can address it later.

Reply:
Thank you for the useful suggestion 🙏 . I fix it at https://github.com/nimblehq/nimble-crypto-ios/pull/23/commits/4772cc0eb884ce8ace1972269fdcedc9490b2673.